docs/usage/labels.md

```diff
@@ -0,0 +1,125 @@
+VALUE is a sequence of characters with length from 1 to 255 characters.  
+The query separator character (|) must be escaped with a backslash (\) if it is present.
+For array values, the separator between the values in the array is ||.  
+Delimiter between the operator and its operands is exactly one whitespace: ' '
+Example:
+x in [val1||val2]|y = 5|z eqornil value with \| separator
+```
+
+## Operators
+
+* Equals (**=**):
+    - Checks whether the left operand's value and the right operand are equal
+    - Example: `platform_id = my_platform_id`
+* Not equals (**!=**):
+    - Checks whether the left operand's value and the right operand are not equal
+    - Example: `platform_id != my_platform_id`
+* Equals or nil (**eqornil**)
+    - Checks whether the left operand's value is equal to the right operand OR the left operand's value is NULL
+    - Example: `platform_id eqornil my_platform_id`
+* Greater than (**gt**):
+    - Checks whether the left operand's value is greater than the right operand. Supports only numerical values.
+    - Example: `id gt 5`
+* Less than (**lt**)
+    - Checks whether the left operand's value is less than the right operand. Supports only numerical values.
+    - Example: `id lt 5`
+* In (**in**)
+    - Checks whether the left operand's value is contained in the right operand. Works only for list values of the right operand contained in square braces.
+    - Example: `id in [5|6|7]`
+* Not in (**notin**)
+    - Checks whether the left operand's value is NOT contained in the right operand. Works only for list values of the right operand contained in square braces.
+    - Example: `id notin [1|2|3]`
+
+## Query Types
+
+Queries let you select resources based on the value of either the resource fields or the labels attached to the resource (or both).
+
+* Field Query  
+A field query is a query that is performed on the fields of the object.  
+Example: The `visibility` object has the field `platform_id` so one might say `Give me all visibilities for a platform with id 038001bc-80bd-4d67-bf3a-956e4d545e3c`. This translates to `GET /visibilities?fieldQuery=platform_id = 038001bc-80bd-4d67-bf3a-956e4d545e3c`
+
+* Label Query  
+A label query is a query that is performed on the labels associated with the object.
+Example: You might label multiple visibilities with the label `test = true` saying that this is test data. So getting all non-test visibilities (these are the ones that either have `test = false` or they don't have a `test` label) would translate to `GET /visibilities?labelQuery=test eqornil false`
+
+* Mixed Query  
+A mixed query is a query that is performed both on fields and labels.  
+Example: `Give me all non-test visibilities for platform with id 038001bc-80bd-4d67-bf3a-956e4d545e3c.` This would translate to `/visibilities?fieldQuery=platform_id = 038001bc-80bd-4d67-bf3a-956e4d545e3c&labelQuery=test eqornil false`
+
+# Supported entities
```

Comment:
In some places we use entities and in other resources - should we unify to resources